scripts/globals/effects/astral_conduit.lua

```diff
@@ -0,0 +1,18 @@
+-----------------------------------
+--
+--     tpz.effect.ASTRAL_CONDUIT
+--     
+-----------------------------------
+
+function onEffectGain(target, effect)
+target:addMod(tpz.mod.BP_DELAY, 99)
```

Comment:
A couple of these scripts have improper indentation, the ones I see right now are this one and asylum.lua

Reply:
reformatted indentation